frontend/src/Student/components/OtherMediaSubmissionForm.js

```diff
@@ -199,6 +210,8 @@ class OtherSubmissionForm extends Component {
             }),
             title: yup.string().required('Required'),
             comment: yup.string(),
+            hometown: yup.string().required('Required'),
+            displayName: yup.string().required('Required'),
```

Comment:
Isn't this also conditional on the checkbox

Reply:
addressed in newest commit

Reply:
Displayname still showing up as required for me

Reply:
Fixed